packages/waku/src/lib/utils/path.ts

```diff
@@ -36,8 +36,17 @@ export const fileURLToFilePath = (fileURL: string) => {
 };
 
 // for filePath
```

Comment:
joinPath is only for "filePath" style. Please do not modify for multiple path styles. We should adjust it in advance. This makes things more maintainable.